boskos/resources.json

```diff
@@ -1,5 +1,822 @@
 [
-{"name": "FAKE_PROJECT_1", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_2", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_3", "type": "project", "state": "free"}
+  {
```

Comment:
when adding new projects it does not do a blind reload, it will compare existing and new projects, add new ones and delete old ones respectively.